commit a0812c180ff1026bb201b662bd2c3f8c11fbfa21
Author: Jay Logue <jaylogue@google.com>
Date:   Thu Aug 29 20:17:41 2019 -0700

    openweave-esp32-lwip-patch-0005 : Fix ip6_get_subnet_id()
    
    This change corresponds to commit 192ac1284fa1b24d8cb9cd770eb8b4ece234fdc1 from tps/lwip:nest/stable
    
            Author: Abtin Keshavarzian <abtink@nestlabs.com>
            Date:   Fri Apr 8 09:35:44 2016 -0700
    
            ip6_addr.h: Fix the ip6_get_subnet_id() marco to return correct
            subnet id
    
            This commit changes the macro definition of ip6_get_subnet_id() in
            ip6_addr.h header file, so it returns the correct subnet id for a
            given ipv6 address.

diff --git a/lwip/src/include/lwip/ip6_addr.h b/lwip/src/include/lwip/ip6_addr.h
index 381b237..5544469 100644
--- a/lwip/src/include/lwip/ip6_addr.h
+++ b/lwip/src/include/lwip/ip6_addr.h
@@ -157,7 +157,7 @@ typedef struct ip6_addr ip6_addr_t;
                                     ((addr1)->addr[2] == (addr2)->addr[2]) && \
                                     ((addr1)->addr[3] == (addr2)->addr[3]))
 
-#define ip6_get_subnet_id(ip6addr)   (lwip_htonl((ip6addr)->addr[2]) & 0x0000ffffUL)
+#define ip6_get_subnet_id(ip6addr)   (lwip_htonl((ip6addr)->addr[1]) & 0x0000ffffUL)
 
 #define ip6_addr_isany_val(ip6addr) (((ip6addr).addr[0] == 0) && \
                                      ((ip6addr).addr[1] == 0) && \
